OrbitGl/GpuTrack.cpp

```diff
@@ -20,6 +21,9 @@ GpuTrack::GpuTrack(TimeGraph* time_graph,
   max_time_ = std::numeric_limits<TickType>::min();
 
   string_manager_ = string_manager;
+
+  // Gpu tracks are collapsed by default.
+  collapse_toggle_.SetActive(false);
```

Comment:
This feels a bit like the state is the opposite of what I would it expect to be. I read "collapse toggle active" as "UI element is collapsed", but here it's the inverse.

Reply:
I agree, I changed it to an enum which is much more explicit.